modules/atomic-widgets/assets/js/editor/commands/styles.js

```diff
@@ -43,6 +43,8 @@ export class Styles extends $e.modules.editor.document.CommandHistoryDebounceBas
 		} );
 
 		container.model.set( 'styles', isRedo ? changes.new.styles : changes.old.styles );
+
+		$e.run( 'document/atomic-widgets/styles-updated', { container } );
```

Comment:
Seems like `settings` command uses in restore the same command:
<img width="809" alt="image" src="https://github.com/user-attachments/assets/8ebf3b14-057d-4044-ad2d-8084cbd6d5d4">